lib/cli.js

```diff
@@ -18,6 +18,7 @@ let options = {
     hot: false,
     port: 8080,
     verbose: false,
+    headers: {},
```

Comment:
I'd remove this since it doesn't do anything and Object.assign accepts undefined as an argument.